### PR TITLE
Issue #351: Sort flows by enabled and name

### LINKED ISSUE #351: Bug when enabling / disabling the order of flows changes
The order should always be:
* [ ] first by enabled and then disabled in two groups
* [ ] then by alphabetical, ignoring case

### PR DESCRIPTION
**Pull request by petersistrom**
Resolves #351

### CODE REVIEW

**Comment by jaypha** [comment]
I get an error.

`Exception - Call to protected method flexible_table::set_sorting_preferences() from context 'tool_dataflows\visualiser'`

```
Stack trace:

    line 191 of /admin/tool/dataflows/classes/visualiser.php: Error thrown
    line 63 of /admin/tool/dataflows/index.php: call to tool_dataflows\visualiser::display_dataflows_table()
```

File: classes/dataflows_table.php [hunk 1]
```diff
@@ -216,6 +220,20 @@ public function col_config(\stdClass $record): string {
...
 
+    /**
+     * Display if the dataflow is enabled
+     *
+     * @param \stdClass $record
+     * @return string
+     */
+    public function col_enabled(\stdClass $record): string {
+        if ($record->enabled) {
+            return 'Y';
```
Comment (brendanheywood) [inline classes/dataflows_table.php:231]: This needs to be translated, so either an icon (preferred) or a yes / no

Comment (jaypha) [reply classes/dataflows_table.php:231]: Is this even needed? There is already is enabled/disabled icon in the action column.

Comment (petersistrom) [reply classes/dataflows_table.php:231]: The column is required in order for the table to sort by enabled. One option would be to move the enable/disable button from the action column and have it in this column

Comment (jaypha) [reply classes/dataflows_table.php:231]: OK.

**Comment by jaypha** [comment]
I find that clicking on 'name' does not change the sort priority.
screenie

File: classes/dataflows_table.php [hunk 2]
```diff
@@ -72,6 +72,14 @@ public function make_columns(): void {
         $this->define_headers($headers);
     }
 
+    /**
+     * Set sql sort value
+     *
+     */
+    function get_sql_sort() {
```
Comment (jaypha) [inline classes/dataflows_table.php:79]: It appears you forgot to put in 'public'.
You also need to add a @return to the phpdoc.

**Pull request merged**